miner/miner.go

```diff
@@ -447,11 +447,12 @@ func (m *Miner) computeTicket(ctx context.Context, brand *types.BeaconEntry, bas
 		return nil, xerrors.Errorf("failed to marshal address to cbor: %w", err)
 	}
 
-	if !haveNewEntries {
+	round := base.TipSet.Height() + base.NullRounds + 1
+	if round > build.UpgradeSmokeHeight {
 		buf.Write(base.TipSet.MinTicket().VRFProof)
 	}
 
```

Comment:
will this change break the previous manner here if we upgrade to 0.7.0 right now? 
say, if we are under `build. UpgradeSmokeHeight`, and do not have new entries, the vrf would be written into buffer in 0.6.2 [https://github.com/filecoin-project/lotus/blob/v0.6.2/miner/miner.go#L450-L452](https://github.com/filecoin-project/lotus/blob/v0.6.2/miner/miner.go#L450-L452)

Reply:
The `haveNewEntries` was always true as drand period is equal to filecoin period so every block was guaranteed to have at least one beacon entry.

Reply:
good to know that~

> The `haveNewEntries` was always true as drand period is equal to filecoin period so every block was guaranteed to have at least one beacon entry.

glad to know that~